catalog/transformer/catalog-transformer-pptx/src/main/java/ddf/catalog/transformer/input/pptx/PptxInputTransformer.java

```diff
@@ -225,17 +222,14 @@ private void extractThumbnail(Metacard metacard, InputStream input) throws IOExc
                         IMAGE_QUALITY);
                 return outputStream.toByteArray();
             }
-        } catch (RuntimeException e) {
-            if (e.getCause() instanceof javax.imageio.IIOException) {
-                LOGGER.debug("unable to generate thumbnail for PPTX file", e);
-            } else {
-                throw e;
-            }
+        } catch (IOException | RuntimeException e) {
+            LOGGER.debug("Unable to generate thumbnail for PPTX file", e);
```

Comment:
Instead of passing the `IOException` to the caller, catch it here and return `null`